client/blocks/product-purchase-features/product-purchase-features-list/jetpack-return-to-dashboard.jsx

```diff
@@ -4,14 +4,18 @@
 import React from 'react';
 import { localize } from 'i18n-calypso';
 import { get } from 'lodash';
+import unTrailingSlashIt from 'lib/route/untrailingslashit';
```

Comment:
Nitpicking: the `lib/route` utilities are exported in `./index` too, so you can import them shorter like this:

```
import untrailingslashit from 'lib/route';
```